core/js/typespecs/api_types.ts

```diff
@@ -372,6 +373,7 @@ export const experimentEventSchema = z
     span_parents: experimentEventBaseSchema.shape.span_parents,
     root_span_id: experimentEventBaseSchema.shape.root_span_id,
     span_attributes: experimentEventBaseSchema.shape.span_attributes,
+    is_root: experimentEventBaseSchema.shape.is_root,
```

Comment:
We should be able to check for the presence of this in our internal REST api unit tests.